third_party/README.md

```diff
@@ -23,3 +23,8 @@ When importing source code from upstream libyuv, the following changes must be d
 4. The commit hash of libyuv from where the files got imported in this README.md file must be updated.
 5. `LIBYUV_API` must be removed from any and all imported functions as these files are always built static.
 6. Replace .cc file extension by .c.
+
+## libwebp
```

Comment:
I thought it was just for traceability: so that we do not have diverging codebases for unicode handling (without knowing it), putting it inside third_party highlights that any modification should probably be backported upstream. Then again, we can just add a comment on top of the file.